src/plugins/saved_objects/README.md

```diff
@@ -0,0 +1,119 @@
+# Saved Object
+
+The saved object plugin provides all the core services and functionalities of the saved objects, and is utilized by many other plugins such as visualizaton plugin, dashboard plugin and wizard plugin etc.
```

Comment:
```suggestion
The saved object plugin provides all the core services and functionalities of saved objects, and is utilized by many core plugins such as `visualizations`, `dashboard`,  and `wizard`, as well as external plugins.
```

This might also be a good place to mention some of the basic mechanics of the saved object system.